File: 1915-check-if-one-string-swap-can-make-strings-equal/check-if-one-string-swap-can-make-strings-equal.py
class Solution:
    def areAlmostEqual(self, s1: str, s2: str) -> bool:
        pos = []
        for i in range(len(s1)):
            if s1[i] != s2[i]:
                pos.append(i)
        if len(pos) == 2:
            return (s1[pos[0]] == s2[pos[1]] and s2[pos[0]] == s1[pos[1]])
        if len(pos) == 0:
            return True
        return False

File: 1915-check-if-one-string-swap-can-make-strings-equal/check-if-one-string-swap-can-make-strings-equal.js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {boolean}
 */
var areAlmostEqual = function (s1, s2) {
    let pos = []
    for (let i = 0; i < s1.length; i++) {
        if (s1.charAt(i) != s2.charAt(i)) {
            pos.push(i)
        }
    }
    if (pos.length == 2 || pos.length == 0) {
        return (s1.charAt(pos[0]) == s2.charAt(pos[1]) && s2.charAt(pos[0]) == s1.charAt(pos[1]))
    }
    return false
};